pkg/discovery/discovery.go

```diff
@@ -45,7 +45,9 @@ func Run(kubeClient kubernetes.Interface, cfg *config.Config) (errCount uint) {
 	metapath := path.Join(outpath, MetaLocation)
 	err := os.MkdirAll(metapath, 0755)
 	if err != nil {
-		panic(err.Error())
+		errCount++
+		errlog.LogError(errors.Wrap(err, "could not create directory to store results"))
+		return errCount
```

Comment:
can just be `return errCount++` then lgtm

Reply:
That's what I tried initially but it seems go does not support that. TIL: https://play.golang.org/p/FjcRXGMMDfz

Reply:
grumble golang ... `return errCount+1`

Reply:
Ahh! Brain fart on my end. Will update.